static/app/components/contextData/index.tsx

```diff
@@ -113,12 +127,25 @@ function walk({
             preserveQuotes,
             withAnnotatedText,
             jsonConsts,
-            meta: meta?.[i]?.[''] ?? meta?.[i] ?? meta?.[''] ?? meta,
+            meta: getChildMetaContainer(meta, i),
```

Comment:
The other way to fix https://github.com/getsentry/sentry/pull/55202/files#r1304930204 is to bring back `?? meta`.